cmake/Package.cmake

```diff
@@ -0,0 +1,23 @@
+# -- packaging -----------------------------------------------------------------
+
+set(CPACK_PACKAGE_VENDOR "Tenzir")
```

Comment:
We might wanna put Tenzir GmbH here, not sure exactly.

Reply:
"Tenzir" should be enough.